pipeline/train/configs/opustrainer/teacher.one-stage.cjk.yml

```diff
@@ -0,0 +1,37 @@
+datasets:
```

Comment:
This could use a few sentences of documentation at the top to explain why this config exists, and why you would use it compared to other configs.